blocks/log.py

```diff
@@ -64,10 +69,3 @@ def previous_row(self):
     @property
     def last_epoch_row(self):
         return self[self.status['_epoch_ends'][-1]]
-
-    def to_dataframe(self):
-        """Convert a log into a :class:`.DataFrame`."""
-        if not PANDAS_AVAILABLE:
-            raise ImportError("The pandas library is not found. You can"
-                              " install it with pip.")
-        return DataFrame.from_dict(self, orient='index')
```

Comment:
Pandas has advantages over my one-liner, but I was trying to show that it is definitely possible to inspect logs without `to_dataframe`. Hence, IMHO it is not essental enough to justify 20 lines of code and documentation in order to bring down the amount of code the user needs to type from 41 characters to 19 characters. What ever happened to lean and mean? `blocks-extras` seems like a much better place for these kind of non-essential but handy shortcuts. We could have a `log_converter` module in there that has `to_dataframe`, `to_csv`, etc.

And "lean and mean" isn't just out of principle. Introducing these kind of "small dependencies" you argue for means more code that can break because of changes in external interface, installations that can be slow or fail, and frustration from users that commands that they rightfully expect to work end with an `ImportError`.

I also think 24 hours is a perfectly good turnaround time for a PR. If we are going to wait days so that every single person of the development team can ponder things over, development will come to a crawl. It often takes 1-2 days just for a single person to have a look, if every single one of us needs to give the LGTM we will be waiting weeks to merge 5-line PRs. If there is something you insist on reviewing, I would suggest leaving a comment saying that you have concerns and ask merging to be delayed (like you did with the new serialization method).